tools/pyboard.py

```diff
@@ -237,7 +237,7 @@ def __init__(self, cmd):
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
         )
-        pty_line = self.subp.stderr.readline().decode("utf-8")
+        pty_line = self.subp.stdout.readline().decode("utf-8")
```

Comment:
Is there any chance this might break some other process that prints the PTY to stderr not stdout?

(Maybe fixable by changing `stderr=subprocess.STDOUT` in the call to Popen? Although if it's not something that anyone uses then might as well ignore.)

Reply:
I was wondering why this was stderr to begin with... it looks like this commit used it: a9e6f08adb6dbe84ee59e9545895aa9dfc5ade17 , but that's since been removed due to zephyr moving to cmake.


(Note, this change is also needed by the qemu-arm port.)

Reply:
I've updated to use `stderr=subprocess.STDOUT`.